library/src/lib/alert/alert-ref.ts

```diff
@@ -0,0 +1,13 @@
+import { Observable, Subject } from 'rxjs';
+
+export class AlertRef {
+    
+    private readonly _afterDismissed: Subject<undefined> = new Subject<undefined>();
+    public afterDismissed: Observable<undefined> = this._afterDismissed.asObservable();
```

Comment:
is `close` is the output of the fd-alert, should is also be called close here? (or `onClose`)

Reply:
I'll make the close event onDismiss